src/main/java/viper/gobra/frontend/GobraParserBase.java

```diff
@@ -0,0 +1,28 @@
+// Imported to Gobra from https://github.com/antlr/grammars-v4/blob/4c06ad8cc8130931c75ca0b17cbc1453f3830cd2/golang
+
+package viper.gobra.frontend;
+import java.util.List;
+import org.antlr.v4.runtime.*;
+import viper.gobra.frontend.GobraLexer;
+
+/**
+ * All parser methods that are used in grammar (p, prev, notLineTerminator, etc.)
+ * should start with lower case char similar to parser rules.
+ */
+public abstract class GobraParserBase extends org.antlr.v4.runtime.Parser
+{
+    protected GobraParserBase(TokenStream input) {
+        super(input);
+    }
+
+    /**
+     * Returns true if the current Token is a closing bracket (")" or "}")
```

Comment:
```suggestion
     * Returns true if the current Token is a closing bracket (")" or "}" or "!>")
```

Reply:
Currently, "!>" is a closing bracket. If the method returns true if the current token is a closing bracket, then this includes "!>". Alternatively, the method is doing something else. I understand that where this function is used, to use it to check for the semicolon before a closing bracket, but this is orthogonal to the description of this method.